nodebuilder/core/config.go

```diff
@@ -19,13 +19,23 @@ type Config struct {
 	IP       string
 	RPCPort  string
 	GRPCPort string
+	// TLSEnabled specifies whether the connection is secure or not.
+	// PLEASE NOTE: it should be set to true in order to handle TLSPath and/or XTokenPath.
+	TLSEnabled bool
```

Comment:
Do we need this if we just provide a TLSPath?

Reply:
Yes, we need it for a use case when we have to provide an empty config.